### DISCUSSION TITLE
Start and enable live timing option right mouse

### DISCUSSION THREAD

**Discussion by DoeEensGek** [f1multiviewer/issue-tracker#104] [Feature requests] [answered]
Small feature request. Possible to start live timing with right mouse click on onboard. (And maybe enable disable for specific onboard, but thats additional.)

Great progress and superb app!

**Comment by f1multiviewer** [comment]
Oh good one, you can already enable/disable by pressing `T` btw :)

**Comment by jeppeolesen** [comment, marked as answer]
Closing this as there is the `T` keyboard shortcut to cycle through the live timing overlays on onboards.
